frontend/src/components/object/ObjectTable.vue

```diff
@@ -117,7 +119,6 @@ const loadLazyData = (event?: any) => {
       tableData.value = r.data;
       totalRecords.value = +r?.headers['x-total-rows'];
       loading.value = false;
-
       // add to object store
       r.data.forEach((o: COMSObject) => {
         objectStore.$patch((state) => {
```

Comment:
Not a change from your PR, but we should avoid using globals like this and instead turn this into a store action.